lib/options.js

```diff
@@ -145,6 +145,15 @@ module.exports = {
           'in the request/response body.',
         external: true,
         usage: ['VALIDATION']
+      },
+      {
+        name: 'Enable strict request matching',
+        id: 'strictRequestMatching',
+        type: 'boolean',
+        default: false,
+        description: 'Whether schema operations be strictly matched with collection request or not.',
```

Comment:
the `or not` is redundant
Also, the requests are matched with operations, not the other way round
`Whether requests should be strictly matched with schema operations. Setting to true will not include any matches where the URL path segments don't match exactly.`